projects/plugins/jetpack/extensions/blocks/sharing-button/sharing-button.php

```diff
@@ -116,6 +120,9 @@ function render_block( $attr, $content, $block ) {
 	);
 	$component .= $style_type !== 'text' ? $icon : '';
 	$component .= '<span class="jetpack-sharing-button__service-label" aria-hidden="true">' . esc_html( $title ) . '</span>';
+	if ( $service_name === 'share' ) {
+		$component .= '<span class="tooltiptext">' . esc_html__( 'Copied to clipboard', 'jetpack' ) . '</span>';
```

Comment:
This doesn't seem to be accessible via screen readers, alongside the "display none" handling. I'm not sure how to best approach this though. Maybe @monsieur-z could help here?

Reply:
It doesn't work indeed. Tags with the `aria-live` attribute read their content when it is added, but here the text `Copied to clipboard` is present from the get-go. It should be added to the `span` after the button is clicked`.

Since the text is rendered visually, an alternative to using `aria-live` could be adding `role="status"` to the span instead (see [this article](https://www.sarasoueidan.com/blog/accessible-notifications-with-aria-live-regions-part-1/)). Any of the 2 implementations would be fine, I guess.